### DISCUSSION TITLE
Changing the font in the code area

### DISCUSSION THREAD

**Discussion by pravin** [beekeeper-studio/beekeeper-studio#996] [Q&A]
Is there any way to change the font used in the code area? I'm on a HiDPI display and the default font doesn't look that great. I currently zoom in -- which is not ideal as it zooms the whole interface.

Any config files that I can edit to change it?

**Comment by jlanzarotta** [comment]
This question has not been answered. I am interested in changing the default font as well and would be interested in an answer.
